pkg/storage/replica.go

```diff
@@ -252,11 +252,15 @@ type Replica struct {
 	RangeID roachpb.RangeID // Should only be set by the constructor.
 
 	store *Store
-	// sha1 hash of the system config @ last gossip. No synchronized access;
-	// must only be accessed from maybeGossipSystemConfig (which in turn is
-	// only called from the Raft-processing goroutine).
-	systemDBHash []byte
-	abortCache   *AbortCache // Avoids anomalous reads after abort
+
+	// sha1 hash of the system config @ last gossip. Must be accessed only
+	// from the Raft-processing goroutine.
+	systemConfigHash []byte
+	// sha1 hash of node livenesses @ last gossip. Must be accessed only
+	// from the Raft-processing goroutine.
+	nodeLivenessHash []byte
```

Comment:
Given that both the system config span and node liveness span should exist in only a single range, I think we could move these fields to `Store`. Not terribly important, but it is somewhat weird that they exist here as only a single range in the system will need them.

Btw, do we ensure the node liveness span is not split as we do with the system config span?